src/views/AnnualReport.vue

```diff
@@ -526,40 +541,44 @@ export default {
   methods: {
     ...mapActions(['setFilingData']),
 
-    fetchData () {
+    async fetchData (): Promise<void> {
       const url = `businesses/${this.entityIncNo}/filings/${this.filingId}`
-      axios.get(url).then(response => {
+      await axios.get(url).then(async response => {
         if (response && response.data) {
           const filing = response.data.filing
           try {
             // verify data
             if (!filing) throw new Error('Missing filing')
             if (!filing.header) throw new Error('Missing header')
+            const header = filing.header
```

Comment:
The diff below looks worse than it is. I simply created header/business/annualReport to simplify the code below.